nile-backend/src/main/java/com/tp/Nile/models/Product.java

```diff
@@ -27,15 +27,15 @@
     private Integer productId;
 
     @ManyToOne(fetch = FetchType.EAGER,
-            cascade = { CascadeType.PERSIST })
+            cascade = { CascadeType.MERGE })
```

Comment:
I was saying to add the merge property along with the persist property. Not delete it.

Reply:
Just tried that, still getting detached entity error in Postman